projects/fsastorefrontlib/src/cms-components/sync-pilot/sync-pilot-connection.component.ts

```diff
@@ -44,11 +46,11 @@ export class SyncPilotConnectionComponent implements OnInit, OnDestroy {
   }
 
   setConfigurationForSyncPilot() {
-    this.syncPilotService.setConfig({
-      stompUrl:
-        'https://msg.dev.livecontract.net/beraterpoolServer/beraterpoolWS',
-      serverUrlServer:
-        'https://msg.dev.livecontract.net/beraterpoolServer/beraterpool/server/v1',
+    this.componentData.data$.subscribe(data => {
+      this.syncPilotService.setConfig({
+        stompUrl: data.stompUrl,
+        serverUrlServer: data.url,
+      });
```

Comment:
In this component you should separate `connect` and `enterQueue` form `connectToSyncPilot` method into two independent methods. Not every component which extends generic needs to call `enterQueue` method while establishing a connection.